Docs/source/usage/parameters.rst

```diff
@@ -717,7 +717,8 @@ Particle initialization
       ``<species_name>.charge`` (`double`) optional (default is read from openPMD file) when set this will be the charge of the physical particle represented by the injected macroparticles.
       ``<species_name>.mass`` (`double`) optional (default is read from openPMD file) when set this will be the charge of the physical particle represented by the injected macroparticles.
       ``<species_name>.z_shift`` (`double`) optional (default is no shift) when set this value will be added to the longitudinal, ``z``, position of the particles.
-      ``<species_name>.impose_t_lab_from_file`` (`bool`) optional (default is false) only read if warpx.gamma_boost > 1., it allows to set t_lab for the Lorentz Transform as being the time stored in the openPMD file.
+      ``<species_name>.t_lab_from_file`` (`bool`) optional (default is false) only read if warpx.gamma_boost > 1., it allows to set t_lab for the Lorentz Transform as being the time stored in the openPMD file.
+      ``<species_name>.t_lab`` (`double`) optional (default is 0) which allows to set a custom t_lab for the Lorentz Transform. This option overrides ``t_lab_from_file`` if the latter is defined.
```

Comment:
Just a comment. Would it work to have this setup where if `t_lab` is not supplied, it means that it will be obtained from the file (instead of having the extra input parameter flag to specify that)?